mobile/lib/services/rest_api.dart

```diff
@@ -165,36 +165,34 @@ class AirqoApiClient {
     return json.decode(response.body)['status'] as bool;
   }
 
-  Future<InsightData> fetchInsightsData(String siteId) async {
+  Future<List<Forecast>> fetchForecast(String siteId) async {
     try {
-      final now = DateTime.now();
-      final utcNow = now.toUtc();
-      final startDateTime = utcNow.getFirstDateOfCalendarMonth().toApiString();
-      final endDateTime = '${DateFormat('yyyy-MM-dd').format(
-        utcNow.getLastDateOfCalendarMonth(),
-      )}T23:59:59Z';
-
-      final queryParams = <String, dynamic>{
-        'siteId': siteId,
-        'utcOffset': now.getUtcOffset(),
-        'startDateTime': startDateTime,
-        'endDateTime': endDateTime,
-      };
+      final body =
+          await _performGetRequest({}, "${AirQoUrls.forecast}/$siteId");
```

Comment:
@Mnoble-19 expecting the to be developed daily forecast endpoint to support querying by siteId.